router/router.go

```diff
@@ -142,6 +147,7 @@ func loadConfig() {
 	retryTimeWindow = config.GetDuration("Router.retryTimeWindowInMins", time.Duration(180)) * time.Minute
 	minRetryBackoff = config.GetDuration("Router.minRetryBackoffInS", time.Duration(10)) * time.Second
 	maxRetryBackoff = config.GetDuration("Router.maxRetryBackoffInS", time.Duration(300)) * time.Second
+	maxFailedListCount = config.GetInt("Router.failedListCount", 10)
```

Comment:
We use size more often, let's do the same.
`Router.failedEventsCacheSize`

Reply:
done